crates/polars-ops/src/chunked_array/strings/namespace.rs

```diff
@@ -15,11 +13,6 @@ use super::*;
 #[cfg(feature = "binary_encoding")]
 use crate::chunked_array::binary::BinaryNameSpaceImpl;
 
-fn f_regex_extract<'a>(reg: &Regex, input: &'a str, group_index: usize) -> Option<Cow<'a, str>> {
```

Comment:
Yes, nice cleanup. Better to have the implementation separated from the dispatch. :+1: